server/integration_tests/explore_test.py

```diff
@@ -370,6 +370,11 @@ def test_e2e_sdg(self):
     ],
                                 dc='sdg')
 
+    self.run_detect_and_fulfill('e2e_sdg_mainDc', [
```

Comment:
super nit: ok for this, but not a fan of mixing snake and camel :)

Reply:
done